CHANGELOG.asciidoc

```diff
@@ -3,6 +3,140 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.10.2]]
+=== Beats version 7.10.2
+https://github.com/elastic/beats/compare/v7.10.1...v7.10.2[View commits]
+
+==== Breaking changes
```

Comment:
Please remove the empty sections.

Reply:
I was waiting for reviews, in case something was missing, will remove before merging